app/serializers/base_serializer.py

```diff
@@ -12,8 +15,13 @@ def fields(*names):
     return data
 
 
+def camel_case(string):
```

Comment:
Since I can't read regex well, can we comment what this does. Like, does it remove all special chars, or just dashes? I feel like if I want to know what this does I should be able to read the function comment and not need to google the regex.

Reply:
There are tests!